ferrilata-devilline.IntegrationTests/Scenarios/ApiGetPitchesTests.cs

```diff
@@ -0,0 +1,84 @@
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using ferrilata_devilline.IntegrationTests.Fixtures;
+using ferrilata_devilline.Models;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace ferrilata_devilline.IntegrationTests.Scenarios
+{
+    [Collection("BaseCollection")]
+    public class IntegrationTests
+    {
+        private readonly TestContext _testContext;
+
+        public IntegrationTests(TestContext testContext)
+        {
+            _testContext = testContext;
+        }
+
+        [Theory]
+        [InlineData("/api/pitches")]
+        public async Task PitchesApi_Should_Return401_WhenRequestHeaderAuthorizationIsMissing(string url)
+        {
+            //Arrange
+            var request = new HttpRequestMessage(HttpMethod.Get, url);
+
+            //Act
+            var response = await _testContext.Client.SendAsync(request);
+
+            //Assert
+            Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
+        }
+
+        [Theory]
+        [InlineData("/api/pitches")]
+        public async Task PitchesApi_Should_ReturnErrorMessage_WhenRequestHeaderAuthorizationIsMissing(string url)
+        {
+            //Arrange
+            var request = new HttpRequestMessage(HttpMethod.Get, url);
+            var expected = new Error("Unauthorizied");
+
+            //Act
+            var response = await _testContext.Client.SendAsync(request);
+            var responseString = await response.Content.ReadAsStringAsync();
+            var actual = JsonConvert.DeserializeObject<Error>(responseString);
+
+            //Assert
+            Assert.True(actual.Equals(expected));
```

Comment:
you have Assert.Equal, Assert.True(A == B) and Assert.True(A.Equal(B)) in here, it would be more elegant with a single version (i know about the bug where sometimes it gets confused and checks the pointers, but we can always deserialize things into strings, as an options, and two identical strings always have the same memory address)

Reply:
It turned out that Assert.Equal calls .Equals under the hood anyway.
Changed all to Assert.Equal.